modules/logger/parser/parser.go

```diff
@@ -164,13 +164,15 @@ func parseAndStoreTestOutput(
 				previousTestName = "summary"
 				logWriter.writeLog(logger, "summary", data)
 
+			case isResultLine(data):
+				// We ignore result lines, because that is handled specially below.
```

Comment:
This fixes the bug where the results lines leak into unrelated test logs. The result lines were ending up in the base case right below this because there was no case guard that handled the result lines separately.